### PR TITLE
Game test

### PR DESCRIPTION
**Pull request by brian-greeson**
**Finished the game tests**
 - SimpleCov says 100% coverage.  
 - Also added SimpleCov.start in the test_helper.rb as per the simplecov usage directions

### CODE REVIEW

File: DTR.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+We'll have a daily standup at 8:45am.
+We'll have group check in and work time from 8pm - 9pm.
+
+
+### Communication Expectations (How and often will we communicate? How do we keep lines of communication open? How will we make decisions as a team?):
+
+Our group expectation is that during times you're committed to working on the project, slack response is expected within 5 minutes -- at the very least, you should send a quick message to let the sender know when you will respond.
+
+Tony prefers text messaging. We will all share our phone numbers but we do all expect good phone etiquette -- no calling or texting after 9pm.
```
Comment (tony-gomes) [inline DTR.md:38]: Slack works for me actually. We can update this on the next retro.

File: lib/game.rb [hunk 2]
```diff
@@ -0,0 +1,36 @@
+class Game
+  @@games = {}
+
+  def self.add(game)
```
Comment (tony-gomes) [inline lib/game.rb]: so much cleaner with variable change!

File: test/game_test.rb [hunk 3]
```diff
@@ -0,0 +1,70 @@
+require "./test/test_helper"
+
+require "pry"
```
Comment (tony-gomes) [inline test/game_test.rb]: pry require already in the test helper, and let’s clip line 2.

File: lib/team.rb [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  @@teams = {}
+
+  def self.add(team)
+    @@teams[team.team_id] = team
+  end
+
+  def self.all
+    @@teams
+  end
+  attr_reader :team_id,
```
Comment (tony-gomes) [inline lib/team.rb]: let’s add a line be 10 & 11 to stay consistent.

File: test/game_test.rb [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+    assert_equal "Toyota Stadium", Game.all[2012030221].venue
+    assert_equal "/api/v1/venues/null", Game.all[2012030221].venue_link
+  end
+
+  def test_it_loads_all_games_from_csv
+    assert_equal 2012030221, Game.all[2012030221].game_id
+    assert_equal 2014030316, Game.all[2014030316].game_id
+
+  end
+
```
Comment (tony-gomes) [inline test/game_test.rb:66]: extra line break here

File: test/test_helper.rb [hunk 6]
```diff
@@ -0,0 +1,6 @@
+require "simplecov"
+SimpleCov.start
+require "minitest/autorun"
+require "mocha/minitest"
+require "minitest/pride"
+require "CSV"
```
Comment (tony-gomes) [inline test/test_helper.rb:6]: we can add the pry require here rather than individual files for convenience.

**Review by tony-gomes — COMMENTED** [review]
killin’ it @brian-greeson!

there are a few typos like extra/needed line breaks, and an extra require pry that we can just keep inside the helper file.

we should also follow ruby convention of an empty line at the end of every file.

however, I am stoked to be working with this data structure ya’ll set up! brilliant.

File: DTR.md (hunk 1 above)
Comment (brian-greeson) [reply DTR.md:38]: I'm really confused on how this file got tracked by git.. i'll remove it from this branch as it's not relevant

File: lib/game.rb (hunk 2 above)
Comment (brian-greeson) [reply lib/game.rb]: So much more we can do to clean it up.. but that can wait until the next refactor.

File: test/test_helper.rb (hunk 6 above)
Comment (brian-greeson) [reply test/test_helper.rb:6]: agreed

**Review by tony-gomes — APPROVED** [review]
Great work @brian-greeson!

**Review by DavidTTran — APPROVED** [review]
Looks good to me, but some of those style/formatting issues might also exist on my branch.

**Review by kristastadler — APPROVED** [review]
Looks good to me, @brian-greeson

**Pull request closed without merging**